src/component/list.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import ListItem from "./listitem";
+
+
+class List extends React.Component {
+
+
+    render() {
+        const { todo, onDelete, onEdit } = this.props;
+
+        return (
+            <>
+                <div className="row justify-content-center  ">
+                    <div className="col-sm-4 bg-secondary pb-3 shadow-lg rounded py-4">
+                        {
+                            todo.map((item) => {
+                                console.log(item)
+                                return <ListItem
+                                    item={item.todo}
+                                    key={item.id}
+                                    handleDelete={() => { onDelete(item.id) }}
```

Comment:
here we have to pass function that should be defined in render() upwards return
const handleDelete = () =>{ onDelete(item.id)}  // defination
handleDelete={handleDelete} // passing